<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>createwindow</class>
    <widget class="QDialog" name="createwindow">
        <property name="windowModality">
            <enum>Qt::ApplicationModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>320</width>
                <height>200</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>20</y>
                    <width>301</width>
                    <height>170</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <spacer name="verticalSpacer_2">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>Virtual Machine Name:</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <spacer name="verticalSpacer_3">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <widget class="QLineEdit" name="name_entry"/>
                        </item>
                    </layout>
                </item>
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                    <widget class="QLabel" name="label_2">
                        <property name="enabled">
                            <bool>false</bool>
                        </property>
                        <property name="text">
                            <string>Slashes and whitespace are not allowed,
                                they will be replaced with a hypen.
                            </string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="ok_btn">
                                <property name="focusPolicy">
                                    <enum>Qt::StrongFocus</enum>
                                </property>
                                <property name="text">
                                    <string>&amp;OK</string>
                                </property>
                                <property name="icon">
                                    <iconset>
                                        <normalon>:/resources/ok_white_36dp.svg</normalon>
                                    </iconset>
                                </property>
                                <property name="autoDefault">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="cancel_btn">
                                <property name="text">
                                    <string>&amp;Cancel</string>
                                </property>
                                <property name="icon">
                                    <iconset>
                                        <normalon>:/resources/close_white_36dp.svg</normalon>
                                    </iconset>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <tabstops>
        <tabstop>name_entry</tabstop>
        <tabstop>ok_btn</tabstop>
        <tabstop>cancel_btn</tabstop>
    </tabstops>
    <resources>
        <include location="../resources/linbox.qrc"/>
    </resources>
    <connections/>
</ui>
